tests/test_refunds.py

```diff
@@ -0,0 +1,86 @@
+PAYMENT_ID = 'tr_7UhSN1zuXS'
+REFUND_ID = 're_4qqhO89gsT'
+
+
+def test_list_all_refunds(client, response):
+    """Retrieve a list of all refunds"""
+    response.get('https://api.mollie.com/v2/refunds', 'refunds_multiple')
+    refunds = client.refunds.all()
+    assert refunds.count == 1
+    assert refunds.__class__.__name__ == 'List'
+
+    iterated = 0
+    iterated_refund_ids = []
+    for refund in refunds:
+        assert refund.__class__.__name__ == 'Refund'
+        iterated += 1
+        assert refund.id is not None
+        iterated_refund_ids.append(refund.id)
+    assert iterated == refunds.count
+    assert len(set(iterated_refund_ids)) == refunds.count
+
+
+def test_list_all_refunds_of_payment(client, response):
+    """Retrieve a list of all refunds of a payment"""
+    response.get('https://api.mollie.com/v2/payments/%s/refunds' % PAYMENT_ID, 'refunds_multiple')
+    response.get('https://api.mollie.com/v2/payments/%s' % PAYMENT_ID, 'payments_create')
+    payment = client.payments.get(PAYMENT_ID)
+    refunds = client.payments.refunds(payment).all()
```

Comment:
Dit klopt niet, voor alle refunds van een payment doe je `client.payments.get(ID).refunds`. Dit is geen logisch codepad